### PR TITLE
427 - Radio Basic Documentation

### PR DESCRIPTION
**Pull request by arecuenco-dsgn**

### CODE REVIEW

File: website/src/categories/components/radio-group/usage.md [hunk 1]
```diff
@@ -8,106 +8,126 @@ eleventyNavigation:
 <section class="no-heading">
```
Comment (anna-lach) [inline website/src/categories/components/radio-group/usage.md:9]: I'm not sure what's going on, but when I'm running the website locally everything looks fine, but here in the preview there is sth wrong with the radio component detail page:

image

**Review by anna-lach — CHANGES_REQUESTED** [review]

File: website/src/categories/components/radio-group/usage.md (hunk 1 above)
Comment (Diaan) [reply website/src/categories/components/radio-group/usage.md:9]: That is code from the `code` tab 🤔

**Review by jpzwarte — APPROVED** [review]

File: website/src/ts/components/install-info.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import { customElement, property } from 'lit/decorators.js';
+
+@customElement('ds-install-info')
+export class InstallInfo extends LitElement {
+  @property() package?: string;
+
+  override render(): TemplateResult {
+    return html`
+      <section>
+        <h2>Installation</h2>
```
Comment (anna-lach) [inline website/src/ts/components/install-info.ts:11]: Installation should contain a link like other headings, right now it's not rendered in the navigation on the right:

image

File: website/src/categories/components/component-table.njk [hunk 3]
```diff
@@ -7,7 +7,7 @@
             <section>
             <h2 id={{ tagName.selector }} tabindex="-1">
               <a class="header-anchor" href="#{{ tagName.selector }}">
-                {%- if component.data.componentTagName.length > 1 -%}{{ tagName.name }}{% endif %} API
+                {%- if component.data.componentTagName.length > 1 -%}{{ tagName.name }}{%- endif -%} API
```
Comment (anna-lach) [inline website/src/categories/components/component-table.njk:10]: Space is missing between component name and API? :)

image



and 


image

**Review by anna-lach — APPROVED** [review]

**Pull request merged**